server/UniverseGenerator.cpp

```diff
@@ -875,18 +875,20 @@ void InitEmpires(const std::map<int, PlayerSetupData>& player_setup_data) {
         if (color_it != colors.end())
             colors.erase(color_it);
 
+        constexpr EmpireColor CLR_ZERO{0, 0, 0, 0};
+
         // if no colour already set, do so automatically
-        if (empire_colour == GG::CLR_ZERO) {
+        if (empire_colour == CLR_ZERO) {
             if (!colors.empty()) {
                 // take next colour from list
                 empire_colour = colors[0];
                 colors.erase(colors.begin());
             } else {
                 // as a last resort, make up a colour
-                empire_colour = GG::FloatClr(static_cast<float>(RandZeroToOne()),
-                                             static_cast<float>(RandZeroToOne()),
-                                             static_cast<float>(RandZeroToOne()),
-                                             1.0f);
+                empire_colour = {RandInt(0, 255),
+                                 RandInt(0, 255),
+                                 RandInt(0, 255),
+                                 255};
```

Comment:
https://ci.appveyor.com/project/freeorion/freeorion/builds/35875836/job/78bhaug75kmcby0b#L275

Reply:
Should be fixed with `static_cast`.